### PR TITLE
give more control over electron build version

### PR DESCRIPTION
**Pull request by bridiver**

### CODE REVIEW

**Comment by bbondy** [comment]
Please update this accordingly, and ping when done, thanks!
https://github.com/brave/browser-laptop/blob/master/docs/electronPrebuilt.md

**Comment by bridiver** [comment]
this seemed to fit better in the readme instructions for npm build-package so I updated it there

File: package.json [hunk 1]
```diff
@@ -4,7 +4,9 @@
   "description": "Brave laptop and desktop browser",
   "main": "./app/index.js",
   "config": {
-    "port": "8080"
+    "port": "8080",
+    "electron_mirror": "https://github.com/brave/electron/releases/download/v",
```
Comment (bbondy) [inline package.json]: this makes browser-laptop-releases obsolete right?

Comment (bridiver) [inline package.json]: to be honest I"m not sure where browser-laptop-releases is configured to be used, but I think it makes it obsolete either way

Comment (bbondy) [inline package.json]: https://github.com/brave/electron-prebuilt/commit/96eebc9f30a8d86c89efe57bd407a5dbe52e7bd5

Comment (bbondy) [inline package.json]: I think it uses the local one once installed from that.

**Comment by luixxiul** [comment]
Did we abandon this?

**Comment by bbondy** [comment]
I think it's still valid but I need to understand this aspect more first in the comments of this PR about browser-laptop-releases.  Not a high priority though relative to other things.

**Comment by willy-b** [comment]
+1 to this PR. It is kind of difficult right now to test different versions of Electron (or one's own dev version).

**Comment by bridiver** [comment]
working on some other changes to make this easier, but this is what I use for now on mac
`script/build.py -c Debug && rsync -avz --delete  out/D/Brave.app ../browser-laptop/node_modules/electron-prebuilt/dist/`

**Comment by willy-b** [comment]
Thanks @bridiver for sharing the command. 
I was also getting a permission issue with an S3 bucket you guys pull from during execution of the `build.py` script, see issue #2899 (https://github.com/brave/browser-laptop/issues/2899)

**Pull request closed without merging**